src/libclient/CMakeLists.txt

```diff
@@ -0,0 +1,175 @@
+project(libowncloudclient)
```

Comment:
lib must not use the name owncloud (due to theming), let's rather use libui or something more descriptive (analogous to `libsync`).